cmsl1t/analyzers/offline_met_analyzer.py

```diff
@@ -58,11 +58,11 @@
 
 def ExtractSums(event):
     offline = dict(
-        HTT=Sum(event.sums.caloHt),
+        HTT=Sum(event.sums.Ht),
```

Comment:
Note to myself: this kind of stuff is addressed in #80 where it should become possible to select these on config level